### PR TITLE
Add deploy configuration

### PR DESCRIPTION
**Pull request by bigfootjon**
Still to be done:
- [x] Setup some URL to ping against (instead of my website):

The URL needs to support the following:
- [x] Be publicly accessible (i.e. circleci.com's servers can hit it)
- [ ] Able to pull the monorepo AND build AggieMap (& and trees?)
- [x] Access the aggiemap.tamu.edu FTP server

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -23,3 +23,21 @@ jobs:
       - run:
           name: Run Prettier
           command: npx prettier **/*.ts --check
+  deploy:
+    machine:
+      enabled: true
+    steps:
+      - run:
+          name: Signal Deploy Ready
+          command: curl https://jonjanzen.com/test/deploy?sha=${CIRCLE_SHA1}
```
Comment (Bigfootjon) [inline .circleci/config.yml]: This URL needs to be changed

**Comment by Bigfootjon** [comment]
@danielgoldbergtamu Is there some public service that I could add an URL to?

**Comment by Bigfootjon** [comment]
Per offline discussion, Edgar and I have a plan:

* Ask Nathan to install git and npm on the aggiemap server
* Create an ASP page that runs a bat script to deploy
* For security, setup a shared secret that CircleCI will include in the request to the ASP page, which the ASP page will then verify

**Comment by bigfootjon** [comment]
Still to be done:

* [ ] Ask Nathan to fix the npm config?
* [ ] Create a directory somewhere to run the deploy script

**Comment by bigfootjon** [comment]
Closing this to clear out my PR queue. Let me know if you need help with this.

**Pull request closed without merging**